lib/oban/pruner.ex

```diff
@@ -71,11 +71,14 @@ defmodule Oban.Pruner do
   end
 
   defp prune_beats(conf) do
+    start_mono = System.monotonic_time(:microsecond)
```

Comment:
What do you think about putting this in the `Oban.Telemetry` module as a helper function? It could have a signature like this:

```elixir
@spec span(name :: atom(), fun :: (-> result :: term())) :: result
```

Something to standardize the measurement and naming would be nice.

Reply:
Agreed